definitions.md

```diff
@@ -0,0 +1,50 @@
+# Definition of Common Terms
+
+## Infrastructure
+
+- <a name="hub"/>**Hub**:
+    - **Definition**: A kubernetes cluster that is running OCM, typically connected to one or more [spokes](#spoke).
+    - Common alternatives to avoid:
+        - Control plane
+        - Control cluster
+- <a name="spoke"/>**Spoke**:
+    - **Definition**: A Kubernetes cluster connected to a [hub](#hub) where resources can be placed via a manifestwork.
+    - Common alternatives to avoid:
+        - Workload cluster
+        - Cluster
+        - Gateway cluster
+        - Data plane
+        - Managed cluster
+- <a name="workload"/>**Workload**:
+    - **Definition**: A set of resources deployed to one or more [spokes](#spoke) - typically accessed via a [gateway](#gateway) - and designed to complete a task, e.g. an API or web-page.
+    - Common Alternatives to avoid:
+        - Application
+        - API
+        - deployment
+        - backend
+- <a name="gateway"/>**Gateway**:
+    - **Definition**:
+        - On a [spoke](#spoke): A resource designed to facilitate ingress to a [workload](#workload)
+        - On a [hub](#hub): A resource which follows specific mutation and replication rules to be replicated into [spokes](spokes).
+
+## Personas
+- <a name="gateway-admin"/>**Gateway Admin**
+    - **Definition**: A user who is primarily concerned with configuring gateways across one or more [spokes](#spoke)
+    - Common alternatives to avoid:
+        - Sysadmin
+        - Admin
+        - SRE
+        - cluster-admin
+- <a name="cluster-admin" />**Cluster admin**:
+    - **Definition**: A user who is primarily concerned with the configuration of a particular [workload](#workload) connected to a [gateway](#gateway) on a [spoke](#spoke).
```

Comment:
> configuration of a particular [workload]

This sounds like the concern of the application developer.
Would the cluster operator persona in gateway api docs align with what you're thinking for cluster admin?

Reply:
yes, ok. I meant this more as "how a workload is connected to a gateway" and maybe need to add some messaging around "ensuring traffic is flowing to the workload correctly"